modules/product/product.resolver.js

```diff
@@ -0,0 +1,14 @@
+const productsService = require('./product.service');
+
+const productsQuery = {
+  getProductsById: (parent, args) => productsService.getProductsById(args.id),
```

Comment:
Please, check if the document is found and throw an error if the product is not found. Look at the news resolver and its error handling.

Reply:
Error handling will be in the next pull request